readme.md

```diff
@@ -100,7 +100,7 @@ const items = ['item', 'item2'];
 </ul>;
 ```
 
-### Helper templates
+### Custom templates
```

Comment:
I would leave the name "Helper templates" here and specify that there are 2 ways of creating a helper template. Calling a typescript function as a function and calling it like a an element.